.travis.yml

```diff
@@ -8,7 +8,9 @@ python:
 install:
     - sudo apt-get install graphviz || sudo apt-get update && sudo apt-get install graphviz
     - pip install coverage coveralls
+    - pip install flake8
     - pip install -e .
+before_script: "make lint"
```

Comment:
Fixed before_script.

I'll put them all on one line for now, but can do the requirements file in a follow up request.